### PR TITLE
Translate 2016 Fukuoka Ruby Award Competition (ja)

### PR DESCRIPTION
**Pull request by ryoben**
please review.

### CODE REVIEW

File: ja/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+GMOペパボ賞 賞品：
+
+* ムームードメインで使用できる５万円分クーポン
+* 地元の食品やおやつの詰め合わせ（３万円相当）
+
+IIJ GIO賞 賞品：
+
+* IIJ GIOで使用できる50万円分クーポン (最大6ヶ月)
+
+”Matzがあなたのコードをテストし、レビューするなんて、とても貴重です！しかも、応募は無料です。”
```
Comment (marocchino) [inline ja/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: Not a big part but, [in other jp docs](https://github.com/ruby/www.ruby-lang.org/blob/master/ja/news/_posts/2002-11-28-20021128.md). Matz translated with 'まつもとさん'

**Comment by ryoben** [comment]
replace "Matz" with "まつもとさん"

**Comment by stomar** [comment]
Please squash commits and I'll merge.

**Comment by ryoben** [comment]
@stomar  please merge

**Comment by stomar** [comment]
:+1: Thanks.

**Pull request merged**